frontend/src/metabase-lib/filter.ts

```diff
@@ -632,6 +632,7 @@ function isExcludeDateBucket(
 
 const DATE_FORMAT = "yyyy-MM-DD";
 const TIME_FORMAT = "HH:mm:ss";
+const TIME_FORMAT_MS = "HH:mm:SS.sss";
```

Comment:
For compatibility with existing format https://github.com/metabase/metabase/blob/1f7e261ec4702b36aa1f815440396187c4b7e931/frontend/src/metabase/query_builder/components/filters/pickers/TimePicker/TimePicker.jsx#L23, tests TBD in another PR